### ISSUE TITLE
After new database is added in source_daimon it is not shown

### ISSUE THREAD

**Issue by PRijnbeek** [OHDSI/WebAPI#1862]
### Expected behavior
When i add a new database to source and source_daimon http://localhost:8080/WebAPI/source/sources would show it and it would be visible in Atlas

### Actual behavior
It is not

### Steps to reproduce behavior
Add new and test

I tried the refresh under configuration that does not help.

reload the WebAPI in Tomcat solves the problem, this should not be needed

**Comment by chrisknoll** [comment]
We can add a call to refresh (endpiont: /WebAPI/sources/refresh) after the new source is added.  I don't mean to say the client should call /refresh, but rather the function that the endpint points to should be invoked after a new source/source_daimon is added.

**Comment by anthonysena** [comment]
This type of information would be useful to note in the documentation. While it is certainly not ideal to have to restart WebAPI after adding a source, we should note that in the documentation (with a link to this issue) so there is awareness.

**Comment by davidhcar** [comment]
Is it expected for refresh url(/WebAPI/sources/refresh) to return 404 status ?

**Comment by chrisknoll** [comment]
No, a 404 shouldn't be returned from /WebAPI/sources/refresh.   Per the original issue, now that we have a config screen for sources, it would make sense to have a refresh button for the sources there that calls out to the /WebAPI/sources/refresh endpoint.

**Comment by chrisknoll** [comment]
So we are caching the data source list now, and the button 'clear server cache' will clear the cache in the event the source is added via INSERTs into the DB.  Depends on #2393
